source/speech/speech.py

```diff
@@ -2376,7 +2377,7 @@ def getFormatFieldSpeech(  # noqa: C901
 		link=attrs.get("link")
 		oldLink=attrsCache.get("link") if attrsCache is not None else None
 		if (link or oldLink is not None) and link!=oldLink:
-			text=_("link") if link else _("out of %s")%_("link")
+			text = _("link") if link else _("%s end") % _("link")

```

Comment:
- This should have a translation comment
- I think it makes sense to put these all into categories to prevent collisions in translation msgID
- I think it's a bad idea to construct the message like this, especially given other "end" messages are not constructed. EG "comment end". This should either enforce for all that the message is ordered this way, or give translators the ability to order it correctly for their language.